yt/yt/server/master/cell_master/program.h

```diff
@@ -201,13 +214,27 @@ class TCellMasterProgram
         _exit(0);
     }
 
+private:
+
+    void PrintCompatibilityInfoAndExit() const {
```

Comment:
For functions and classes, `{` are put onto the next line (and for ifs and fors no):

```
void MaybeExit(bool exit)
{
    if (exit) {
        _exit(0);
    }
}
```